<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileExportDialogBase</class>
 <widget class="QDialog" name="ProfileExportDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export selected profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mPointDistanceLabel">
     <property name="text">
      <string>Point distance</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="mPointDistanceSpinBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mMaximumElevationDifferenceLabel">
     <property name="text">
      <string>Maximum value difference</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="mMaxValueDifferenceSpinBox"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="mRasterLayerComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mRasterLayerLabel">
     <property name="text">
      <string>Raster layer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mOutputFileLabel">
     <property name="text">
      <string>Output file</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mOutputFileLineEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="mOutputFileToolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProfileExportDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfileExportDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
